### ISSUE TITLE
Version 0.43.0 is wonkey af

### ISSUE THREAD

**Issue by duff2013** [motioneye-project/motioneye#2538]
I've been using Motioneye for many years now with my axis network cameras and it worked great (old version python2). My server died so I had to do complete reinstall of motioneye so I saw that it moved to new maintainers that are keeping it up to date now. But it's not stable at all now. I added my cameras but they show and go then away on admin screen. 
<img width="1722" alt="Screen Shot 2022-06-19 at 5 59 06 PM" src="https://user-images.githubusercontent.com/5598200/174507931-43e7b5ee-1d66-43d0-90ba-f7d6c756db04.png">

I also noticed the old version had an "advanced" section for Networked Cameras that is gone now in the web config. 

Also my server (raspberry pi Debian Bullseye Lite) now crashes after I start using motioneye and I have to pull the power to restart it after adding my cameras.

I hope motioneye get back the same stability as python 2 version.

**Comment by peterhoeg** [comment]
We're very much in anecdote territory now, but I've been using the py3 version for over a year (multiple different versions of course) without any issues at all. So if we need to get to the bottom of what's going on here, we all need additional details/logs.

**Comment by Marijn0** [comment]
>So if we need to get to the bottom of what's going on here, we all need additional details/logs.

Yes, without any logs provided we can't help you further. Closing this for now since it's been several years without a response.
